src/openrct2/paint/track/coaster/LatticeTriangleTrack.cpp

```diff
@@ -18417,7 +18417,7 @@ static void LatticeTriangleTrackDiagBrakes(
             session, supportType.metal, kDiagSupportPlacement[direction], 0, height, session.SupportColours);
     }
 
-    PaintUtilSetSegmentSupportHeight(session, kSegmentsAll, 0xFFFF, 0);
+    PaintUtilSetSegmentSupportHeight(session, BlockedSegments::kDiagStraightFlat[trackSequence], 0xFFFF, 0);
```

Comment:
```suggestion
    PaintUtilSetSegmentSupportHeight(session, PaintUtilRotateSegments(BlockedSegments::kDiagStraightFlat[trackSequence], direction), 0xFFFF, 0);
```

I think these should be rotated.